src/visual/VisualGroup.js

```diff
@@ -0,0 +1,58 @@
+import joint from 'jointjs';
+
+const cDefaultWidth = 100;
+const cMinHeight = 100;
+const cPixelPerSymbol = 15;
+const cHeightPerPort = 50;
+
+/** Class that provides graphical representation for the Step.
+ * @private
+ */
+export default class VisualGroup extends joint.shapes.devs.Model {
+  constructor(step, opts = {}) {
```

Comment:
Thank you very much for your valuable review it is making development cleaner! 

In this case I did not use the inheritance because in my opinion it is not necessary, so the both classes are doing different (not at all, them looks like to be inherited but it could be a reason of additional problems related to styles, updates, and bbox functions). 

If you do not mind I'll leave them both inherited from **joint.shapes.devs.Model**